src/client/components/definition/definitionData.js

```diff
@@ -15,11 +15,11 @@ module.exports = {
       },
```

Comment:
just a note for your todo list: we should format the math formulas so they're pretty and nice to read